### DISCUSSION TITLE
part-ii/2007-65/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#72] [Question comments]
Part II 2007 Graph Theory: 2.II.17H 

https://questions.tripos.org/part-ii/2007-65/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part is bookwork; $R(K_t)$ is $R(t,t)$ in the usual definition, and we showed that $R(s,t) \leq {s+t-2 \choose t - 1}$. Setting $s=t$ gives the desired result.

For the second part, we need to show that $R(H) > 6$ and $R(H) \leq 7$. To see that $R(H) > 6$, consider a $K_6$ with vertices labelled $1, \dots, 6$. Colour the triangles 123 and 456 blue, and every other edge red. This does not contain a monochromatic copy of $H$.

To see that $R(H) \leq 7$, consider a $K_7$ with vertices labelled $1, \dots, 7$. This graph must contain a monochromatic triangle since $R(K_3) \leq 6$ (this follows from the inequality earlier in the question, and is in fact bookwork on its own). Assume without loss of generality that the triangle 123 is blue. Then to avoid a blue copy of $H$, every edge between the sets $\{1,2,3\}$ and $\{4,5,6,7\}$ must be red. If any of the edges within the set $\{4,5,6,7\}$ are red, we get a red copy of $H$ (e.g. if the edge 45 is red, then we have a red copy of $H$ with triangle 145 and extra vertex 2). But if all of the edges within $\{4,5,6,7\}$ are blue, then this is a blue copy of $K_4$, which contains a blue copy of $H$.

</details>
